setup/launchMeili.js

```diff
@@ -79,31 +65,21 @@ const defaultRankingRules = [
   const processedDataSet = dataProcessing(dataset)
 
   // Add documents batches array
-  const batchedDataSet = batch(processedDataSet, 10000)
+  const batchedDataSet = batch(processedDataSet, 100000)
```

Comment:
```suggestion
  const batchedDataSet = batch(processedDataSet, BATCH_SIZE)
```

This could be a constant at the top of the file to be able to change it easily and avoid magic numbers in the code: 
```
const BATCH_SIZE = 100000
```